src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -21,7 +22,11 @@
  * The main LogicManager of the app.
  */
 public class LogicManager implements Logic {
-    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: ";
+    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data. The following errors has occured: %s";
```

Comment:
Just some minor grammar changes, assuming that only one error follows this message:

```suggestion
    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data. The following error has occurred: %s";
```

To ensure this format string is not used as-is instead of being wrapped in `String.format`, it might also be helpful to rename it to distinguish it from other errors messages, for e.g. `FILE_OPS_ERROR_TEMPLATE` or `FILE_OPS_ERROR_PREFIX`.

Reply:
Good catch, I will follow the formatting name of the classes here!